tutorial.md

```diff
@@ -0,0 +1,212 @@
+# Google Cloud Client Libraries for Rust
+
+This guide explains how to use the Google Cloud Client Libraries for Rust to interact with Google Cloud services in your applications.
+
+> **Note**: These libraries are currently in development and not recommended for production use.
+
+## Before you begin
+
+1. [Install Rust](https://www.rust-lang.org/tools/install)
+2. [Set up a Google Cloud project](https://cloud.google.com/resource-manager/docs/creating-managing-projects)
+3. [Install and initialize the Google Cloud CLI](https://cloud.google.com/sdk/docs/install)
+
+## Installing the client libraries
+
+Add the required dependencies to your `Cargo.toml`:
+
+```toml
+[dependencies]
+google-cloud-secretmanager-v1 = { version = "0.1", features = ["unstable-stream"] }
+tokio = { version = "1.0", features = ["macros", "rt-multi-thread"] }
+```
+
+## Authenticating
+
+### Using Application Default Credentials (Recommended)
+
+```rust
+use google_cloud_secretmanager_v1::client::SecretManagerServiceClient;
+
+async fn init_client() -> Result<SecretManagerServiceClient, Box<dyn std::error::Error>> {
+    // Creates a client using Application Default Credentials
+    let client = SecretManagerServiceClient::new().await?;
+    Ok(client)
+}
+```
+
+### Using Service Account Keys
+
+```rust
+use google_cloud_auth::credentials::CredentialsFile;
+use google_cloud_secretmanager_v1::client::SecretManagerServiceClient;
+
+async fn init_client_with_credentials() -> Result<SecretManagerServiceClient, Box<dyn std::error::Error>> {
+    // Load credentials from a service account key file
+    let credentials = CredentialsFile::new_from_file(
+        "/path/to/service-account-key.json"
+    ).await?;
+    
+    // Initialize client with explicit credentials
+    let client = SecretManagerServiceClient::builder()
+        .with_credentials(credentials)
+        .build()
+        .await?;
+    
+    Ok(client)
+}
+```
+
+## Using the client libraries
+
+### Basic operations
+
+#### Listing secrets
+
+```rust
+use google_cloud_secretmanager_v1::client::SecretManagerServiceClient;
+
+async fn list_secrets(project_id: &str) -> Result<(), Box<dyn std::error::Error>> {
+    // Initialize the client
+    let client = SecretManagerServiceClient::new().await?;
+    
+    // Format the parent resource name
+    let parent = format!("projects/{}", project_id);
+    
+    // List all secrets in the project
+    let response = client
+        .list_secrets()
+        .set_parent(parent)
+        .send()
+        .await?;
+    
+    // Process the response
+    for secret in response.secrets {
+        println!("Secret: {}", secret.name);
+    }
+    
+    Ok(())
+}
+```
+
+### Working with long-running operations
```

Comment:
We have two tutorials dedicated to this:

https://googleapis.github.io/google-cloud-rust/working_with_long_running_operations.html
https://googleapis.github.io/google-cloud-rust/configuring_polling_policies.html

This seems redundant.